src/rest-server/src/controllers/v2/user.js

```diff
@@ -107,6 +107,10 @@ const createUserIfUserNotExist = async (req, res, next) => {
         if (authConfig.groupConfig.groupDataSource === 'ms-graph') {
           forbiddenMessage = forbiddenMessage + `Please contact your admin, and join the AAD group named [ ${authConfig.groupConfig.defaultGroup.externalName} ].`;
         }
+        return res.redirect(req.returnBackURI + '?'+ querystring.stringify({
```

Comment:
should throw error here and handle all kinds of errors in [`getAAD`](https://github.com/microsoft/pai/blob/master/src/rest-server/src/controllers/v2/token.js#L76)